jenkins/hourly_maintenance.py

```diff
@@ -137,13 +140,41 @@ def KillLoopingBash():
             err |= subprocess.call(['sudo', 'kill', '-9', pid])
     return err
 
+
+def DeleteCorruptGitRepos():
```

Comment:
I could potentially install a new plugin on jenkins to do that, but that's scary :(

This function is super fast (find -size 0 exits quickly with nothing most of the time) and hourly maintenance runs pretty often. 

Maybe a way to run the hourly maintenance on all the slaves during boot